resolver/resolver.go

```diff
@@ -444,7 +444,11 @@ func (res *Resolver) handleEmpty(rs *records.RecordGenerator, name string, m, r
 // compressing the message first and truncating it accordingly.
 func reply(w dns.ResponseWriter, m *dns.Msg, setTruncateBit bool) {
 	m.Compress = true // https://github.com/mesosphere/mesos-dns/issues/{170,173,174}
-	if err := w.WriteMsg(truncate(m, isUDP(w), setTruncateBit)); err != nil {
+	// Get the maximum allowed message size.
```

Comment:
I don't think this comment the next are necessary, the comments that live with the function are more than enough.